### DISCUSSION TITLE
can't find new star setting

### DISCUSSION THREAD

**Discussion by manilx** [immich-app/immich#11802] [Q&A]
Updated server to latest and I can't find the "User Settings > App Settings > Star Rating" setting to enable stars....
Looked in t"https://XXX/admin/system-settings" and can't find it anywhere....

**Comment by bo0tzz** [comment]
It's in the user settings. https://my.immich.app/user-settings?isOpen=app-settings

**Comment by manilx** [comment]
Thx! It's in the user/account setting!!
